dali/base/dasds.cpp

```diff
@@ -94,6 +95,10 @@ static unsigned msgCount=(unsigned)-1;
 // # TYPE sds_pending_requests gauge
 // A unsigned scalar should suffice, i.e. never going to have that many pending transactions
 
+static auto pSdsRequestsCount = hpccMetrics::createMetricAndAddToManager<hpccMetrics::CounterMetric>("sds_requests.received", "The total number of Dali SDS requests handled", SMeasureCount);
+static auto pSdsActiveRequests = hpccMetrics::createMetricAndAddToManager<hpccMetrics::GaugeMetric>("sds_requests.active", "Current number of active SDS requests being handled", SMeasureCount);
+static auto pSdsPendingRequests = hpccMetrics::createMetricAndAddToManager<hpccMetrics::GaugeMetric>("sds_requests.pending", "Current number of pending SDS requests", SMeasureCount);
```

Comment:
I'm not sure how useful this metrics are for alerting or scaling.  I am happy for them to be published - because they will probably look pretty in a graph, but they're only a snapshot.  Publishing the counts allows you to work out the average load over the period, rather than the instantaneous load at the point the metrics were gathered.)

Better to publish "dai.sqs_requests.started", "dai.sqs_requests.processed" or "dai.sqs_requests.completed" (should be consistent with nq above, and same pattern should be used for all types of requests processed in the system.